build/go_check.sh

```diff
@@ -26,6 +26,15 @@ if [[ "${f}" ]]; then
   exit 1
 fi
 
+# Check dependencies up-to-date
+go mod download
+d=$(go get -u all 2>&1)
+if [[ "${d}" ]]; then
```

Comment:
Do you need `-n` or something like that to check that the string isn't empty? I don't know if empty strings are falsey in bash.

Reply:
Yea so empty string is evaluated to false, so no need for that. http://www.linuxintro.org/wiki/Babe#empty_strings